op-node/rollup/derive/engine_queue.go

```diff
@@ -683,6 +706,51 @@ func (eq *EngineQueue) forceNextSafeAttributes(ctx context.Context) error {
 	return nil
 }
 
+// tryBackupUnsafeReorg tries to reorg unsafe head to backupUnsafeHead.
+func (eq *EngineQueue) tryBackupUnsafeReorg(ctx context.Context) error {
+	defer func() {
+		// Reset backupUnsafeHead to avoid multiple usage.
+		eq.backupUnsafeHead = eth.L2BlockRef{}
+		// Only try once because execution engine may forgot backupUnsafeHead
+		// or backupUnsafeHead is not part of the chain.
+		eq.checkBackupUnsafeReorg = false
```

Comment:
This will also disable retrying when the RPC request fails with temporary errors(such as network issue). is it intended?

Reply:
Fixed at https://github.com/testinprod-io/optimism/pull/44/commits/4b777debaceda6d6698f40868354e14c2fe2d973, and added tests.